interfaces/Portal/src/app/components/registration-activity-detail/registration-activity-detail.component.html

```diff
@@ -1,4 +1,5 @@
 <article
+  data-testid="activity-detail-tile"
```

Comment:
I think it could be a good practice to reflect the name of the component in the id as much as possible, to avoid confusion. So, in the case of this file, all of the testids would be prefixed with `registration-activity-detail`
```suggestion
  data-testid="registration-activity-detail-tile"
```

This applies to all testid additions

Reply:
Added for all registration testids